[Synthetics] Update infrastructure monitoring for k8s (#6367)
elastic/docs-content: solutions/observability/synthetics/index.md

::::

::::{important}
**Not for infrastructure or {{k8s}} monitoring**

> The {{synthetics-app}} is designed for active synthetic checks against user-defined URLs and user journeys. It is **not** intended for host or pod availability monitoring via autodiscovery (for example, automatically checking every pod in a {{k8s}} cluster as it starts and stops).

Reviewer: ```suggestion
The {{synthetics-app}} is designed for active synthetic checks against user-defined URLs and user journeys, not for infrastructure or {{k8s}} pod uptime monitoring.
```

For infrastructure or {{k8s}} uptime monitoring, use one of the following approaches instead:

> * **{{heartbeat}} with autodiscovery**: Run {{heartbeat}} directly on your infrastructure and use its [autodiscovery](https://www.elastic.co/guide/en/beats/heartbeat/current/configuration-autodiscover.html) capabilities to dynamically monitor hosts and pods. Results appear in the [{{uptime-app}}](/solutions/observability/uptime/index.md).

Reviewer: ```suggestion
* **[{{heartbeat}}](/reference/heartbeat/index.md) with autodiscovery**: Run {{heartbeat}} on your infrastructure and use [autodiscovery](/reference/heartbeat/configuration-autodiscover.md) to dynamically monitor hosts and pods. Results appear in the [{{uptime-app}}](/solutions/observability/uptime/index.md).
```

> * **{{agent}} with the Uptime Monitors integration**: Deploy a standalone {{agent}} and configure the Uptime Monitors ({{heartbeat}}) integration to collect availability data from your infrastructure. Note that the Uptime app is deprecated as of 8.15.

Reviewer: ```suggestion
* **[{{agent}}](/reference/fleet/install-elastic-agents.md) with the Uptime Monitors ({{heartbeat}}) integration**: Deploy {{agent}} and configure the Uptime Monitors integration to collect infrastructure availability data. For [Kubernetes autodiscovery](/reference/fleet/elastic-agent-kubernetes-autodiscovery.md), hints-based autodiscovery requires standalone mode. Note that the Uptime app is deprecated as of 8.15.
```
